src/ui/src/js/controllers/admin_user_view.js

```diff
@@ -73,7 +73,7 @@ export default function adminUserViewController(
     // identifiers for us anyway.
     model.role_assignments.forEach((assignment) => {
       if (assignment.domain.scope === 'Global') {
-        assignment.domain.identifiers.name = '';
+        assignment.domain.identifiers = {name: ''};
```

Comment:
This isn't really related to the main issue, it's just fixing a bug I discovered while testing.